README.md

```diff
@@ -227,6 +227,16 @@ The [fs-extra](https://www.npmjs.com/package/fs-extra) package.
 let content = await fs.readFile('./package.json')
 ```
 
+#### `globby` package
+
+The [globby](https://github.com/sindresorhus/globby) package.
+
+```js
+let packages = await globby(['package.json', 'packages/*/package.json'])
```

Comment:
Also, will it be better for users (more readable) if we export const glob?

Reply:
The package is called `globby` so it can confuse someone.

Reply:
Makes sense.